backend/src/main/resources/application-e2e.yaml

```diff
@@ -0,0 +1,26 @@
+spring:
+  profiles.include: no-security, no-okta-mgmt, server-debug, create-sample-data, local
+  liquibase:
+    user: simple_report_migrations
+    password: migrations456
+  jpa:
+    properties:
+      hibernate:
+        show_sql: true
+        format_sql: false # override this in application-local.yaml to get pretty-printed SQL all the time
+        default_schema: simple_report
+logging:
+  level:
+    # NOTE: add any of the below in application-local.yaml to turn on something interesting
+    # Hibernate SQL query logging: basically the same as show_sql, above, but through slf4j
+    # org.hibernate.SQL: DEBUG
+    # Hibernate input and output value logging: SUPER VERBOSE
+    # org.hibernate.type: TRACE
+    # Always have our own debug logging turned on in dev environments:
+    gov.cdc.usds: DEBUG
+simple-report:
+  data-hub:
+    upload-enabled: ${DATAHUB_UPLOAD_ENABLE:false}
```

Comment:
Is this something we'll want to test e2e at some point or should we just disable it?

Reply:
Nope, I just copied the dev config and removed the `no-okta-auth` profile, but I can probably remove this stuff as well :+1: